### PR TITLE
Fix java 14 url to point to always existing version

### PR DESCRIPTION
**Pull request by ThadHouse**
This should mean we no longer need to update the version every time oracle removes one.

### CODE REVIEW

File: buildSrc/src/main/kotlin/JpackageExec.kt [hunk 1]
```diff
@@ -234,7 +234,7 @@ open class JpackageExec : DefaultTask() {
             fileAssociations.ifPresent { propsFile ->
                 args.addAll("--file-associations", propsFile.asFile.absolutePath)
             }
-            args.addAll("--package-type", installerType.get())
+            args.addAll("--type", installerType.get())
```
Comment (SamCarlberg) [inline buildSrc/src/main/kotlin/JpackageExec.kt]: Why is this change necessary?

Comment (ThadHouse) [reply buildSrc/src/main/kotlin/JpackageExec.kt]: Because they keep changing flags in jpackage. That's why we're going to a fixed version we're hosting.
